client-encryption/src/main/java/io/confluent/kafka/schemaregistry/encryption/FieldEncryptionExecutor.java

```diff
@@ -70,29 +38,19 @@ public class FieldEncryptionExecutor extends FieldRuleExecutor {
 
   public static final String TYPE = "ENCRYPT";
 
-  public static final String ENCRYPT_KEK_NAME = "encrypt.kek.name";
-  public static final String ENCRYPT_KMS_KEY_ID = "encrypt.kms.key.id";
-  public static final String ENCRYPT_KMS_TYPE = "encrypt.kms.type";
-  public static final String ENCRYPT_DEK_ALGORITHM = "encrypt.dek.algorithm";
-  public static final String ENCRYPT_DEK_EXPIRY_DAYS = "encrypt.dek.expiry.days";
+  public static final String ENCRYPT_KEK_NAME = EncryptionExecutor.ENCRYPT_KEK_NAME;
+  public static final String ENCRYPT_KMS_KEY_ID = EncryptionExecutor.ENCRYPT_KMS_KEY_ID;
+  public static final String ENCRYPT_KMS_TYPE = EncryptionExecutor.ENCRYPT_KMS_TYPE;
+  public static final String ENCRYPT_DEK_ALGORITHM = EncryptionExecutor.ENCRYPT_DEK_ALGORITHM;
+  public static final String ENCRYPT_DEK_EXPIRY_DAYS = EncryptionExecutor.ENCRYPT_DEK_EXPIRY_DAYS;
 
-  public static final String KMS_TYPE_SUFFIX = "://";
-  public static final byte[] EMPTY_AAD = new byte[0];
-  public static final String CACHE_EXPIRY_SECS = "cache.expiry.secs";
-  public static final String CACHE_SIZE = "cache.size";
-  public static final String CLOCK = "clock";
+  public static final String KMS_TYPE_SUFFIX = EncryptionExecutor.KMS_TYPE_SUFFIX;
+  public static final byte[] EMPTY_AAD = EncryptionExecutor.EMPTY_AAD;
+  public static final String CACHE_EXPIRY_SECS = EncryptionExecutor.CACHE_EXPIRY_SECS;
+  public static final String CACHE_SIZE = EncryptionExecutor.CACHE_SIZE;
+  public static final String CLOCK = EncryptionExecutor.CLOCK;
 
-  protected static final int LATEST_VERSION = -1;
-  protected static final byte MAGIC_BYTE = 0x0;
-  protected static final int MILLIS_IN_DAY = 24 * 60 * 60 * 1000;
-  protected static final int VERSION_SIZE = 4;
-
-  private Map<DekFormat, Cryptor> cryptors;
-  private Map<String, ?> configs;
-  private int cacheExpirySecs = -1;
-  private int cacheSize = 10000;
-  private Clock clock = Clock.systemUTC();
-  private DekRegistryClient client;
+  private EncryptionExecutor encryptionExecutor = new EncryptionExecutor();
```

Comment:
The previous code was moved to EncryptionExecutor, and is now invoked via an instance of that class.